pages/configuration/configuration-settings.mdx

```diff
@@ -9,32 +9,182 @@ import { Steps } from 'nextra/components'
+
+        </Steps>
+
+    </Tab>
+
+    <Tab>
+    TBD docker-compose
+    </Tab>
+
+    <Tab>
+        Once Memgraph is installed on your Linux machine. 
+Check the status with: 
+``` bash
+systemctl status memgraph
+```
+        
+        If the instance is in a running state, stop the Memgraph instance by running the following command:
+        ```bash
+        sudo systemctl stop memgraph
+        ```
+       The configuration file is available at `/etc/memgraph/memgraph.conf`. Open the configuration file with your favorite text editor.
+        Modify it, save the changes, and start the instance by running the following command:
+
+        ```bash
+        sudo systemctl start memgraph
+        ```
+    </Tab>
+</Tabs>
+
+
+
+### Change configuration settings during runtime
+
+Memgraph contains settings that can be modified during runtime using a Cypher query.
+Some runtime settings are persisted between multiple runs, while others will
+fallback to the value of the command-line argument.
+
+| Setting name               | Description                                                                                   | Persistent between runs |
+|----------------------------|-----------------------------------------------------------------------------------------------|-------------------------|
+| organization.name          | Name of the organization using the instance of Memgraph (used for verifying the license key). | yes                     |
+| enterprise.license         | License key for Memgraph Enterprise.                                                          | yes                     |
+| server.name                | Bolt server name.                                                                             | yes                     |
+| query.timeout              | Maximum allowed query execution time. Value of 0 means no limit.                              | yes                     |
+| log.level                  | Minimum log level. Allowed values: TRACE, DEBUG, INFO, WARNING, ERROR, CRITICAL.              | no                      |
+| log.to_stderr              | Log messages go to `stderr` in addition to `logfiles`.                                        | no                      |
+| cartesian-product-enabled  | Enforces cartesian product operator during query matching.                                    | no                      |
+
+All settings can be fetched by calling the following query:
 
-You can check the current configuration by using the following query (privilege
-level `CONFIG` is required):
 ```opencypher
-SHOW CONFIG;
+SHOW DATABASE SETTINGS;
+```
+
+To check the value of a single setting, you can use a slightly different query:
+
+```opencypher
+SHOW DATABASE SETTING "setting.name";
 ```
 
+If you want to change a value for a specific setting, following query should be used:
+
+```opencypher
+SET DATABASE SETTING "setting.name" TO "some-value";
+```
+
+
+### Changing the configuration settings via file 
```

Comment:
Is it via file or is it via env var?

Reply:
I mean, what's the better wording here, not to be confused with updating existing memgraph config file

Reply:
IMO it is better via file; I could add a `via external file` or `via another config file.` Because the key or env can be used just to apply the changes from another config file.

Reply:
You can leave it as it is